src/main/java/Duke.java

```diff
@@ -1,10 +1,121 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String line = "\t________________________________________";
+    public static Task[] tasks = new Task[100];
+    public static int currentTask = 0;
+
+    public static void takeInput() {
+        Scanner scannerObject = new Scanner(System.in);
+        while (scannerObject.hasNextLine()) {
+            String command = scannerObject.nextLine();
+            if (command.equals("bye")) {
+                bye();
+                break;
+            } else if (command.equals("list")) {
+                list();
+            } else if (command.startsWith("done")) {
+                done(command.split(" ")[1]);
+            } else if (command.startsWith("todo")) {
+                addTodo(command.split(" ", 2)[1]);
+            } else if (command.startsWith("event")) {
+                addEvent(command.split(" ", 2)[1]);
+            } else if (command.startsWith("deadline")) {
+                addDeadline(command.split(" ", 2)[1]);
+            }
+        }
+    }
+
+    public static void bye() {
```

Comment:
a different preferred like printBye would be better since the variable name may be confusing in the above method